Estimator

atlantiswqq opened the pull request.

galeone commented:
In the end, I preferred to work on this today. So here we go! I refactored your code, and right now, the `Model` type has the `EstimatorServe` method, that it can be used to run the inference on a trained model - all with the support for the dictionaries/numpy/pandas input types.

Please have a look at the tests section I wrote about how to use them and let me know if this is how you'd have imagined this feature (I guess yes, it looks *very* similar to your implementation :+1: )

I have also updated the README. Please let me know your thoughts, and thank you for contributing to the project! :raised_hands: 

Close #22

atlantiswqq commented:
hi galeone:
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;I'm sorry that I just saw your email today. I read your refactoring, which is very user friendly for third parties, and the structure looks clearer than I did before. So I think your code is better than mine. Thank you for your efforts. I will keep an eye on this project and wish it more and more perfect.&nbsp;


------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"Paolo Galeone"<notifications@github.com&gt;;
发送时间:&nbsp;2019年11月18日(星期一) 凌晨5:02
收件人:&nbsp;"galeone/tfgo"<tfgo@noreply.github.com&gt;;
抄送:&nbsp;"王启强"<383750993@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [galeone/tfgo] Estimator (#24)




In the end, I preferred to work on this today. So here we go! I refactored your code, and right now, the Model type has the EstimatorServe method, that it can be used to run the inference on a trained model - all with the support for the dictionaries/numpy/pandas input types.
 
Please have a look at the tests section I wrote about how to use them and let me know if this is how you'd have imagined this feature (I guess yes, it looks very similar to your implementation  )
 
I have also updated the README. Please let me know your thoughts, and thank you for contributing to the project! 
 
Close #22
 
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
